### DISCUSSION TITLE
Please add flavors feature to Framework

### DISCUSSION THREAD

**Discussion by yunusemrecetin** [nylo-core/nylo#44] [General] [answered]
When we started a new project with Flutter. Firstly we configured the flavors then we started the development. If possible; the framework must support flavors.

**Comment by agordn52** [comment]
Hi @yunusemrecetin,

You can configure flavours/environments using different `.env` files.

E.g. 
``` dart
flutter run/build --dart-define=ENV_FILE_PATH=.env.production
// or
flutter run/build --dart-define=ENV_FILE_PATH=.env.developing
``` 
I hope that answers your question 👌
You can read more about **env** files [here](https://nylo.dev/docs/4.x/installation#adding-your-env-file).

Anthony

**Reply by xsorifc28** [reply]
This suggested answer does not work as in [support => lib/nylo.dart](https://github.com/nylo-core/support/blob/4.x/lib/nylo.dart#L66) `.env` is being loaded. 

I do not see any usages of `ENV_FILE_PATH` in the code that would load the defined file, am I missing something hree?

**Reply by agordn52** [reply, marked as answer]
Hi @xsorifc28, 

Thanks for highlighting this, it's now fixed in [v4.1.3](https://github.com/nylo-core/framework/releases/tag/v4.1.3) (nylo_framework).

Update your pubspec.yaml file to use the above version.

E.g. 
``` yaml
dependencies:
  ...
  nylo_framework: ^4.1.3
```

Then, check out the new docs on environment flavours [here](https://nylo.dev/docs/4.x/configuration#environment-flavours), hope that helps.

Thanks,
Anthony

**Reply by xsorifc28** [reply]
Thank you,  I will give this a try.
